Source/.editorconfig

```diff
@@ -12,6 +12,7 @@ tab_width = 2
 
 # New line preferences
 end_of_line = lf
+max_line_length = 200
```

Comment:
Oof, that's really long. Isn't 120 more standard these days?

Reply:
My intent was to set it to conceptually infinite to let the programmer decide when to cut lines. Dafny commonly has very long lines, many going even over 200 characters. If you don't set a `max_line_length` value like this then Rider happily starts cutting up lines while you blink.

Reply:
Great, +1 to hold off until we have the mechanism in the build.